src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -60,6 +60,14 @@ using std::auto_ptr;
 
 /* ****************************************************************************
 *
+* OtisTreatFunction - callback signature for treatOntimeintervalSubscriptions
+*/
+typedef void (*OtisTreatFunction)(std::string tenant, BSONObj* bobjP);
```

Comment:
Sorry, I can't.
The parameter is a function (had forgotten about that).
A function is a pointer by definition.

NTC